code/backend/batch/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -204,14 +203,9 @@ def get_all_files(self):
         return files
 
     def upsert_blob_metadata(self, file_name, metadata):
-        if self.auth_type == "rbac":
```

Comment:
Not strictly related to this PR, but we already have a client created in the constructor, therefore we can use it here without needing if/else logic